src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import AccordionItem from "./item/AccordionItem";
+import {AccordionContextProvider} from "./AccordionProvider";
+export type AccordionProps = {
+  allowMultipleExpanded?: boolean;
+  allowZeroExpanded?: boolean;
+  defaultExpanded?: string[];
+  children: React.ReactNode;
+};
+
+function Accordion(props: AccordionProps) {
+  const {allowMultipleExpanded, allowZeroExpanded, defaultExpanded, children} = props;
+
+  return (
+    <AccordionContextProvider
+      allowMultipleExpanded={allowMultipleExpanded}
+      allowZeroExpanded={allowZeroExpanded}
+      defaultExpanded={defaultExpanded}>
+      <div>{children}</div>
```

Comment:
Do we really need this wrapper?

Reply:
With this wrapper, AccordionItems will be on a separate div, but it has no other functionality.
![image](https://user-images.githubusercontent.com/42540003/127648120-efc6a72b-3312-4885-a5c0-fd61268f5282.png)

Reply:
So, don't we need a wrapper `className` for this container?

Reply:
I added `className` for the container and refactored the style.